pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDaoImpl.java

```diff
@@ -193,7 +193,7 @@ public Optional<ScanError> updateKitRequest(KitRequestShipping kitRequestShippin
                 } else {
                     logger.info("Updated kitRequests w/ ddp_label " + kitRequestShipping.getDdpLabel());
                     if (StringUtils.isNotBlank(kitRequestShipping.getBspCollaboratorParticipantId())) {
-                        dbVals.resultValue = new ScanError(kitRequestShipping.getBspCollaboratorParticipantId(),
+                        dbVals.resultValue = new ScanError(kitRequestShipping.getDdpLabel(), null,
```

Comment:
setting error to null

Reply:
So the underlying issue is that the two-arg version of this constructor was seeing `getBspCollaboratorParticipantId()` as an error?

Reply:
Yes it was writing it in the error field.

Reply:
Can you add a test for `kitDaoImpl.updateKitRequest` that covers this case?

Reply:
I updated the PR with this change